cmd/simulator/metrics/metrics.go

```diff
@@ -16,24 +18,29 @@ type Metrics struct {
 	IssuanceToConfirmationTxTimes prometheus.Summary
 }
 
+func NewSummary(name string, help string) prometheus.Summary {
+	return prometheus.NewSummary(prometheus.SummaryOpts{
+		Name:       name,
+		Help:       help,
+		Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
+	})
+}
+
 // NewMetrics creates and returns a Metrics and registers it with a Collector
-func NewMetrics(reg prometheus.Registerer) *Metrics {
+func NewMetrics(prefix string, reg prometheus.Registerer) *Metrics {
 	m := &Metrics{
-		IssuanceTxTimes: prometheus.NewSummary(prometheus.SummaryOpts{
-			Name:       "tx_issuance_time",
-			Help:       "Individual Tx Issuance Times for a Load Test",
-			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
-		}),
-		ConfirmationTxTimes: prometheus.NewSummary(prometheus.SummaryOpts{
-			Name:       "tx_confirmation_time",
-			Help:       "Individual Tx Confirmation Times for a Load Test",
-			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
-		}),
-		IssuanceToConfirmationTxTimes: prometheus.NewSummary(prometheus.SummaryOpts{
-			Name:       "tx_issuance_to_confirmation_time",
-			Help:       "Individual Tx Issuance To Confirmation Times for a Load Test",
-			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
-		}),
+		IssuanceTxTimes: NewSummary(
+			fmt.Sprintf("%stx_issuance_time", prefix),
```

Comment:
shall we use `%s_tx_issuance_time` and remove appended `_` from prefixes? (i.e `subnet_a_` > `subnet_a`)